base/src/org/spin/queue/notification/support/EMailSender.java

```diff
@@ -63,18 +65,31 @@ public void sendNotification(MADNotificationQueue notification) {
 		notification.getRecipients().forEach(recipient -> {
 			MClient client = MClient.get(notification.getCtx(), notification.getAD_Client_ID());
 			String eMailFrom = client.getRequestEMail();
+			String eMailPsw = client.getRequestUserPW();
```

Comment:
@rcapecce Please use natural language to name the variables that makes it easier to read and analyze the code, change msg to message eMailPsw to requestUserPassword

Note the readability of the code is more important than saving a few characters, on the other hand IDEs like Intellij help with this job in a very easy way

Reply:
@e-Evolution ok, I will use variables more readeable